src/RedisCache/RedisCache/ChangeLog.md

```diff
@@ -18,6 +18,7 @@
         - Additional information about change #1
 -->
 ## Upcoming Release
+* Added `RedisVersion` parameter in `New-AzRedisCache`
```

Comment:
`RedisVersion` should also be added to `Set-AzRedisCache`

Reply:
Added